src/platform/unix/mod.rs

```diff
@@ -1033,6 +1058,11 @@ fn is_socket(fd: c_int) -> bool {
 
 // FFI stuff follows:
 
+#[cfg(target_os="linux")]
+unsafe fn memfd_create(name: *const c_char, flags: usize) -> c_int {
+    syscall!(MEMFD_CREATE, name, flags) as c_int
```

Comment:
That's odd... Do you know whether `glibc` just doesn't provide a wrapper *yet*, or it doesn't intend to? (And if so, why?)

I don't think it's a reason to disable it by default though. It certainly would be useful however to have a `--force-posix-shm` flag along the lines of `--force-inprocess`, so we can easily test the code locally and in CI.